hs/src/Reach/AST/Base.hs

```diff
@@ -60,6 +66,9 @@ data ImpossibleError
   = Err_Impossible String
```

Comment:
This PR essentially tells us that `Err_Impossible` should not exist and instead `DLE_Impossible` should take a different constructor that is more specific so we can write real explanations.